src/services/resources/controllers.ts

```diff
@@ -99,6 +101,31 @@ export const captainVolunteers = async (req: Request, res: Response) => {
     return sendResponse(res, response);
 };
 
+// TODO:- we need to define what "logic" is as a template otherwise this is a dangerous pattern IMO
+const getDataForNeigborhood = async (req: Request, res: Response, logic) => {
```

Comment:
If you are going to use this pattern, you should really define the typescript definition for logic so we don't accidentally break this in the future